DEVELOPMENT.md

```diff
@@ -471,14 +471,14 @@ If you need to add a new CRD type, you will need to add:
     - [clusterrole-aggregate-edit.yaml](./config/clusterrole-aggregate-edit.yaml)
     - [clusterrole-aggregate-view.yaml](./config/clusterrole-aggregate-view.yaml)
 1. Add go structs for the types in
-    [pkg/apis/pipeline/v1alpha1](./pkg/apis/pipeline/v1alpha1) e.g
-    [condition_types.go](./pkg/apis/pipeline/v1alpha1/condition_types.go) This
+    [pkg/apis/pipeline/v1beta1](./pkg/apis/pipeline/v1beta1) e.g
+    [condition_types.go](./pkg/apis/pipeline/v1beta1/condition_types.go) This
```

Comment:
please change `condition_types.go` to `task_types.go`, conditions are deprecated and do not exist in the `beta` API.

Reply:
Thank you for reviewing @pritidesai !
I committed : f71a32c